experiments/Molecule_Generation/LMOExperiment.py

```diff
@@ -52,47 +53,16 @@ def __init__(
         self.user_prompt = user_prompt
         self.verification_prompt = verification_prompt
         self.refinement_prompt = refinement_prompt
-        self.max_synth_score = helper_funcs.get_synthesizability(lead_molecule)
+        self.max_synth_score = SMILES_utils.get_synthesizability(lead_molecule)
         self.min_density = helper_funcs.get_density(lead_molecule)
 
-    @charge.hypothesis
-    def canonicalize_smiles(self, smiles: str) -> str:
-        """
-        Canonicalize a SMILES string. Returns the canonical SMILES.
-        If the SMILES is invalid, returns "Invalid SMILES".
-
-        Args:
-            smiles (str): The input SMILES string.
-        Returns:
-            str: The canonicalized SMILES string.
-        """
-        return helper_funcs.canonicalize_smiles(smiles)
-
-    @charge.hypothesis
-    def verify_smiles(self, smiles: str) -> bool:
-        """
-        Verify if a SMILES string is valid. Returns True if valid, False otherwise.
-
-        Args:
-            smiles (str): The input SMILES string.
-        Returns:
-            bool: True if the SMILES is valid, False otherwise.
-        """
-        return helper_funcs.verify_smiles(smiles)
-
-    @charge.hypothesis
-    def get_synthesizability(self, smiles: str) -> float:
-        """
-        Calculate the synthesizability of a molecule given its SMILES string.
-        Values range from 1.0 (highly synthesizable) to 10.0 (not synthesizable).
-
-        Args:
-            smiles (str): The input SMILES string.
-        Returns:
-            float: The synthesizability score.
-        """
+        # Registering some pre-written hypothesis tools
```

Comment:
@szaman19 I like how this is evolving to use the new approach to registering known tool functions.